components/tokens/custom-spectrum/custom-large-vars.css

```diff
@@ -78,8 +78,15 @@ governing permissions and limitations under the License.
   --spectrum-well-padding: 20px;
   --spectrum-well-margin-top: 5px;
   --spectrum-well-min-width: 300px;
-  --spectrum-well-border-radius: 5px;
 
+  --spectrum-well-border-radius: 5px;
+  
+  --spectrum-dialog-confirm-hero-height: 160px; 
+  --spectrum-dialog-confirm-border-radius: 5px;
+  --spectrum-dialog-confirm-title-text-size: 19px;
+  --spectrum-dialog-confirm-description-text-size: 15px;
+  --spectrum-dialog-confirm-padding-400: 24px;
```

Comment:
Can we rename `--spectrum-dialog-confirm-padding-400` throughout as well? It looks like this is the padding that is used for the grid variant? In that case it could be `--spectrum-dialog-confirm-padding-grid`

Reply:
Done!